Update Brazilian Portuguese translation to Aug. 2023 (#146)
thewca/wca-regulations-translations: portuguese-brazilian/wca-regulations.md

- 1a) Uma competição precisa ter um Delegado da WCA e uma equipe de organização (composta por um ou mais membros) com os seguintes oficiais: juízes, embaralhadores e anotadores de resultados.
- 1b) A equipe de organização de uma competição é responsável pela logística antes, durante e depois da competição.
- 1c) O Delegado da WCA é responsável por assegurar que a competição segue o Regulamento da WCA a qualquer política ou requisito da WCA aplicáveis. O Delegado da WCA pode delegar responsabilidades para outros membros da equipe de organização, mas é responsável em última instância pela forma como elas são conduzidas.
    - 1c1) O Delegado da WCA deve garantir que a competição siga a [Política de Confidencialidade de Embaralhamentos da WCA](https://www.worldcubeassociation.org/documents/policies/external/Scramble%20Accountability.pdf).
- 1e) Cada evento precisa ter um ou mais juízes.
    - 1e1) Um juiz é responsável por executar os procedimentos do evento.
        - 1e1a) Um juiz pode fiscalizar vários competidores simultaneamente a critério do Delegado da WCA, contanto que o juiz seja capaz de assegurar que todo o Regulamento da WCA seja seguido em todos os momentos.

>     - 1e2) Todos os competidores tem que estar disponíveis para ser juízes. Se for requisitado para ser juiz, um competidor pode ser dispensado somente por uma razão legítima (p. ex. não ter conhecimento sobre o quebra-cabeça), a critério do Delegado da WCA. Pena: desqualificação de um evento, múltiplos eventos, ou todos os eventos naquela competição  (ver [Regra 2k](regulations:regulation:2k)).

Reviewer: ```suggestion
    - 1e2) Todos os competidores tem que estar disponíveis para ser juízes. Se for requisitado para ser juiz, um competidor pode ser dispensado somente por uma razão legítima (p. ex. não ter conhecimento sobre o quebra-cabeça), a critério do Delegado da WCA. Pena: desqualificação de um evento, múltiplos eventos, ou todos os eventos naquela competição (ver [Regra 2k](regulations:regulation:2k)).
```

- 1f) Cada evento precisa ter um ou mais embaralhadores. Exceção: 3x3x3 em Menos Movimentos.
    - 1f1) Um embaralhador aplica as sequências de embaralhamento para preparar os quebra-cabeças para as tentativas (ver [Regra A2](regulations:regulation:A2)).

>     - 1f2) Todos os competidores tem que estar disponíveis para serem embaralhadores. Se for requisitado para embaralhar, um competidor pode ser dispensado somente por uma razão legítima (p. ex. não conhecer a notação de embaralhamento), a critério do Delegado da WCA. Pena: desqualificação de um evento, múltiplos eventos, ou todos os eventos naquela competição  (ver [Regra 2k](regulations:regulation:2k)).

Reviewer: ```suggestion
    - 1f2) Todos os competidores tem que estar disponíveis para serem embaralhadores. Se for requisitado para embaralhar, um competidor pode ser dispensado somente por uma razão legítima (p. ex. não conhecer a notação de embaralhamento), a critério do Delegado da WCA. Pena: desqualificação de um evento, múltiplos eventos, ou todos os eventos naquela competição (ver [Regra 2k](regulations:regulation:2k)).
```

- 2f) Os competidores têm que obedecer às regras do local e se portar de maneira apropriada.
- 2g) Os competidores precisam permanecer quietos quando estiverem dentro da área designada de competição. É permitido falar, mas o barulho tem que ser mantido em um nível razoável, e distante dos competidores que estiverem competindo ativamente.
    - 2g3) Os competidores em um grupo em progresso não podem se comunicar uns com outros competidores no mesmo grupo sobre os embaralhamentos dos quebra-cabeças do grupo, exceto caso todos estes competidores não tenham mais nenhuma tentativa a realizar neste grupo. Pena: desqualificação do competidor(es) daquele evento, a critério do Delegado da WCA. Exceção: Para 3x3x3 em Menos Movimentos, competidores que já tiverem terminados suas tentativas podem compartilhar a sequência de embaralhamento com qualquer pessoa que não estiver ativamente competindo na mesma tentativa.
- 2h) Os competidores precisam estar totalmente vestidos dentro do local de competição. A critério do Delegado, competidores podem ser desqualificados da competição por se vestir de maneira inadequada.

> - 2i) Enquanto estiverem competindo ou resolvendo, os competidores não podem usar equipamentos eletrônicos ou de áudio (p. ex. celulares, tocadores de MP3, dictafones, iluminação adicional) além do timer Stackmat ou cronômetro. Pena: Desqualificação da tentativa (DNF).

Reviewer: ```suggestion
- 2i) Enquanto estiverem competindo ou resolvendo, os competidores não podem usar equipamentos eletrônicos ou de áudio (p. ex. celulares, tocadores de MP3, dictafones, iluminação adicional) além do timer Stackmat ou cronômetro. Pena: desqualificação da tentativa (DNF).
```

    - 3h2) Quebra-cabeças cujo plástico colorido seja visível na parte interna do quebra-cabeça (p. ex. quebra-cabeças “stickerless”) são permitidos. Isso não inclui os seguintes quebra-cabeças:
        - 3h2a) Quebra-cabeças com partes transparentes que podem revelar mais informações sobre o estado do quebra-cabeça. Exceção: um adesivo sobreposto ou um logotipo (ver [Regra 3l](regulations:regulation:3l)).
    - 3h3) Quaisquer modificações em um quebra-cabeça que resultem em desempenho ruim por um competidor não são razão para tentativas adicionais.
    - 3h4) Para o Clock, as seguintes modificações são permitidas:

>         - 3h4a) "Inserts customizados (do mesmo formato e tamanho que os inserts de papel tradicionais) são permitidos, a critério do Delegado da WCA. Os inserts tem que ter uma clara indicação das 12 horas que coincida com a dos inserts originais.

Reviewer: ```suggestion
        - 3h4a) Inserções customizadas (do mesmo formato e tamanho que as inserções de papel tradicionais) são permitidos, a critério do Delegado da WCA. As inserções tem que ter uma clara indicação das 12 horas que coincida com a das inserções originais.
```

        - 5b5a) Se uma ou mais partes sem faces coloridas forem afetadas, o quebra-cabeça é considerado resolvido.
        - 5b5b) Se uma parte com uma face colorida for afetada, o quebra-cabeça é considerado resolvido.
        - 5b5c) Se mais de uma parte com uma face colorida for afetada, o quebra-cabeça é considerado não resolvido (DNF).
        - 5b5d) Se uma ou mais partes com mais de uma face colorida forem afetadas, o quebra-cabeça é considerado não resolvido (DNF).
        - 5b5e) As Regras [5b5c](regulations:regulation:5b5c) e [5b5d](regulations:regulation:5b5d) sobrepõem-se às [5b5a](regulations:regulation:5b5a) e [5b5b](regulations:regulation:5b5b).
        - 5b5f) Se uma peça estiver parcialmente desconectada ou não posicionada completamente, a sua posição final é a mais próxima posição em que a peça afetada poderia se encaixar normalmente no mecanismo do quebra-cabeça, considerando o estado atual do quebra-cabeça. Se a posição final resultar na peça resolvida, a peça não é considerada como afetada pelo defeito do quebra-cabeça.

>     - 5b6) Definição: Uma parte é um ou mais, totalmente conectados, componentes de um grupo de componentes de um quebra-cabeça que mantêm-se conectados e não se movem em relação uns aos outros durante operações normais em um quebra-cabeça (p. ex. uma tampa de centro de um 3x3x3, uma peça de ponta de um Pyraminx, uma peça de canto de um 3x3x3 faltando apenas uma face, uma face de uma peça de canto de um 3x3x3).

Reviewer: ```suggestion
    - 5b6) Definição: Uma parte é um ou mais componente totalmente conectados de um grupo de componentes de um quebra-cabeça que mantêm-se conectados e não se movem em relação uns aos outros durante operações normais em um quebra-cabeça (p. ex. uma tampa de centro de um 3x3x3, uma peça de ponta de um Pyraminx, uma peça de canto de um 3x3x3 faltando apenas uma face, uma face de uma peça de canto de um 3x3x3).
```

        - 9b5a) Formato de rodada completa para esse evento: "Melhor de X" (onde X é 1, 2 ou 3).
        - 9b5b) Formato de corte para esses eventos: fase de corte de "Melhor de X" (onde X é 1 ou 2) para "Melhor de Y" (onde Y é 2 ou 3, e Y > X).
- 9f) Os resultados de uma rodada são medidos da seguinte forma:
    - 9f1) Todos os resultados cronometrados abaixo de 10 minutos, com exceção de 3x3x3 Multi-Vendado, são truncados para baixo em centésimos de segundo. Todas as médias cronometradas abaixo de 10 minutos são arredondadas para o centésimo de segundo mais próximo.
    - 9f2) Todos os resultados e médias cronometrados acima de 10 minutos, incluindo todos os tempos para resultados de 3x3x3 Multi-Vendado, são arredondados para o segundo mais próximo (p. ex. X.49 se torna X e X.50 se torna X+1).
    - 9f4) O resultado de uma tentativa é registrado como DNF (Did Not Finish) se a tentativa foi desqualificada ou não foi completada.

>         - 9f4a) Se um competidor violar uma regra claramente de modo que a tentativa será, com certeza, desqualificada, o juíz deve interromper imedaitamente a tentativa. Se o juíz estiver incerto ou se uma disputa puder ser disruptiva (p. ex. porque uma tentativa extra poderia atrasar a competição) o juíz deve consultar o Delegado da WCA.

Reviewer: ```suggestion
        - 9f4a) Se um competidor violar uma regra claramente de modo que a tentativa será, com certeza, desqualificada, o juiz deve interromper imedaitamente a tentativa. Se o juiz estiver incerto ou se uma disputa puder ser disruptiva (p. ex. porque uma tentativa extra poderia atrasar a competição), o juiz deve consultar o Delegado da WCA.
```

    - 9m2) Rodadas com 15 competidores ou menos podem ter no máximo uma rodada subsequente.
    - 9m3) Rodadas com 7 competidores ou menos não podem ter rodadas subsequentes.
- 9o) Rodadas de Corte contam como uma rodada no evento.
- 9p) Se um evento tem múltiplas rodadas, então:
    - 9p1) No mínimo 25% dos competidores têm que ser eliminados entre rodadas consecutivas do mesmo evento.
    - 9p2) Os competidores que avançam para a próxima rodada têm que ser determinados por uma posição (melhores X competidores) ou por resultado (todos os competidores com um resultado melhor que X) na rodada anterior.
        - 9p2a) Para cada rodada, as condições de avanço precisam ser anunciadas antes que a rodada comece, e não devem ser alteradas depois disso. Alterações somente podem ser feitas a critério do Delegado da WCA, que precisa considerar cuidadosamente se a mudança é justa.
    - 9p3) Se um competidor classificado se retira de uma rodada, ele pode ser substituído pelo competidor melhor classificado abaixo do corte na rodada anterior.
- 9s) Cada rodada de um evento precisa ter um limite de tempo (ver [Regra A1a](regulations:regulation:A1a)).

> - 9t) Se qualquer mudança ocorrer aos resultados de um competidor após o término da competição (p. ex. erro de anotação de resultados, penalidade retroativa), de modo que este competidor não devesse ter avançado para a rodada seguinte (ou não tenha sido autorizado a participar da rodada), todos os resultados deste competidor de todos as rodadas subsequentes têm que ser removidos. (Ver [Regra 9u](regulations:regulation:9u))

Reviewer: ```suggestion
- 9t) Se qualquer mudança ocorrer aos resultados de um competidor após o término da competição (p. ex. erro de anotação de resultados, penalidade retroativa), de modo que este competidor não devesse ter avançado para a rodada seguinte (ou não tenha sido autorizado a participar da rodada), todos os resultados deste competidor de todos as rodadas subsequentes têm que ser removidos (Ver [Regra 9u](regulations:regulation:9u)).
```

    - 9t1) Se isto fizer com que uma rodada não atenda à [Regra 9p1](regulations:regulation:9p1), resultados de outros competidores não podem ser removidos.
    - 9t2) Se isto fizer com que uma rodada não atenda às [regras 9m1](regulations:regulation:9m1), [9m2](regulations:regulation:9m2) ou [9m3](regulations:regulation:9m3), quaisquer rodadas subsequentes não podem ser removidas.
    - 9t3) Se qualquer mudança ocorrer ao resultado de um competidor, de modo que qualquer outro competidor não devesse ter avançado para a próxima rodada, o resultado do segundo competidor não poderá ser removido.

> - 9u) Uma competição é considerada terminada se todos os requisitos seuintes estiverem atentidos simultaneamente:

Reviewer: ```suggestion
- 9u) Uma competição é considerada terminada se todos os seguintes requisitos estiverem atendidos simultaneamente:
```

    - A3b) O juiz prepara o timer ligando-o e zerando-o se necessário. Se o Timer Geração 5 Pro estiver em utilização, o juiz tem que assegurar que ele está no modo 2-pad. Separadamente, o juiz também prepara um cronômetro para marcar o tempo de inspeção.
        - A3b1) Quando o juiz acredita que o competidor está pronto, ele pergunta "PRONTO?". O competidor tem que estar pronto para iniciar a tentativa dentro de um minuto após ter sido chamado, senão perde o direito à tentativa (DNS), a critério do juiz.
        - A3b2) O competidor inicia a tentativa confirmando que está pronto, e então o juiz descobre o quebra-cabeça e começa a marcar o tempo de inspeção.
        - A3b3) Quando 8 segundos de inspeção tiverem passado, o juíz diz "8 SEGUNDOS".
        - A3b4) Quando 12 segundos de inspeção tiverem passado, o juíz diz "12 SEGUNDOS".
    - A3c) O competidor pode pegar o quebra-cabeça durante a inspeção.

>         - A3c1) O competidor não pode aplicar movimentos ou mudanças no alinhamento intencionais durante a inspeção. Pena: desqualificação da tentativa (DNF). Excelões: veja [Regra A3c2](regulations:regulation:A3c2) e [Regra A3c5](regulations:regulation:A3c5).

Reviewer: ```suggestion
        - A3c1) O competidor não pode aplicar movimentos ou mudanças no alinhamento intencionais durante a inspeção. Pena: desqualificação da tentativa (DNF). Exceção: veja [Regra A3c2](regulations:regulation:A3c2) e [Regra A3c5](regulations:regulation:A3c5).
```

- A4) Iniciando a resolução:
    - A4b) O competidor usa seus dedos para tocar a parte elevada do timer Stackmat. Suas palmas têm que estar para baixo, e posicionadas no lado do timer mais próximo dele. Pena: pena de tempo (+2 segundos).
        - A4b1) O competidor não pode ter contato físico com o quebra-cabeça enquanto estiver começando a resolução. Pena: pena de tempo (+2 segundos).
    - A4d) Se o timer Stackmat estiver sendo usado, o competidor deve manter suas mãos no timer até que ele veja uma luz verde no timer. Ele inicia a resolução removendo suas mãos do timer (e então iniciando o timer).
        - A4d1) O competidor tem que iniciar a resolução dentro de 15 segundos após o início da inspeção. Pena: pena de tempo (+2 segundos).
        - A4d2) O competidor tem que iniciar a resolução dentro de 17 segundos após o início da inspeção. Pena: desqualificação da tentativa (DNF).
        - A4d3) Se um cronômetro estiver sendo usado, o juiz inicia o cronômetro no momento em que o competidor iniciar a resolução.
    - A4e) Pena de tempo para o início da resolução são cumulativas.
- A5) Durante a resolução:

>     - A5a) Enquanto estiver inspecionando ou resolvendo o quebra-cabeça, o competidor não pode se comunicar com ninguém além do juiz ou Delegado da WCA. Pena: desqualificação da tentativa (DNF). Exceção: Se o comeptidor não ganhar nenhuma vantagem por se comunicar com qualquer outra pessoa além do juíz ou do Delegado da WCA, o resultado pode permanecer, a critério do Delegado da WCA.

Reviewer: ```suggestion
    - A5a) Enquanto estiver inspecionando ou resolvendo o quebra-cabeça, o competidor não pode se comunicar com ninguém além do juiz ou Delegado da WCA. Pena: desqualificação da tentativa (DNF). Exceção: Se o competidor não ganhar nenhuma vantagem por se comunicar com qualquer outra pessoa além do juiz ou do Delegado da WCA, o resultado pode permanecer, a critério do Delegado da WCA.
```

    - A6e) Depois de soltar o quebra-cabeça, o competidor não pode tocar, aplicar um movimento, ou qualquer mudança no alinhamento do quebra-cabeça até que o juiz o tenha inspecionado.
        - A6e1) Penalidade para aplicar um movimento ao quebra-cabeça após parar o timer: desqualificação da tentativa (DNF).
        - A6e2) Penalidade para tocar o quebra-cabeça após parar o timer sem aplicar nenhum movimento: pena de tempo (+2 segundos). Exceção: Se, imediatamente após tocar o quebra-cabeça, o competidor brevemente tocar o quebra-cabeça sem afetar o seu estado, a pena de tempo deve ser desconsiderada, a critério do juiz (ver [Regra 11f](regulations:regulation:11f)).
    - A6f) O competidor não pode zerar o timer até que o juiz tenha anotado o resultado na súmula. Pena: desqualificação da tentativa (DNF), a critério do juiz.
        - A6f1) Se o competidor zerar o timer antes que o resultado tenha sido registrado completamente, o juiz não pode escrever o resultado a partir de sua memória ou de evidência fotográfica, e tem que desqualificar a tentativa (DNF).

>     - A6g) O juiz determina se o quebra-cabeça está resolvido. Se o quebra-cabeça estiver desalinhado, este não pode ser tocado antes que o juíz tenha determinado o seu estado final.

Reviewer: ```suggestion
    - A6g) O juiz determina se o quebra-cabeça está resolvido. Se o quebra-cabeça estiver desalinhado, este não pode ser tocado antes que o juiz tenha determinado o seu estado final.
```

    - A6h) Em caso de disputa, nenhum movimento ou alinhamento pode ser aplicado ao quebra-cabeça até que a disputa seja resolvida.
    - A6i) Penas de tempo para a parte de terminar a resolução são cumulativas.
- A7) Registrando resultados:
    - A7a) O juiz diz ao competidor o resultado.

>         - A7a1) O juíz deve indicar ao competido que o quebra-cabeça está resolvido, ou que uma penalidade será aplicada.

Reviewer: ```suggestion
        - A7a1) O juiz deve indicar ao competido que o quebra-cabeça está resolvido, ou que uma penalidade será aplicada.
```

    - A7b) O juiz anota o resultado em uma súmula e assina próximo a teste para atestar que o resultado está correto, completo, formatado corretamente e legível.
        - A7b1) Se alguma pena for aplicada, o juiz anota o resultado original mostrado no timer, juntamente com quaisquer penas. O formato deve ser "X + T + Y = F", em que X representa a soma das penas de tempo antes/no início da resolução, T representa o tempo mostrado no timer, Y representa a soma das  de tempo durante/depois da resolução, e F representa o resultado final (p. ex. 2 + 17,65 + 2 = 21,65). Se X e/ou Y forem 0, os termos 0 são omitidos (p. ex. 17,65 + 2 = 19,65).
    - A7c) O competidor precisa verificar o resultado registrado e assinar (ou colocar sua inicial) a súmula para reconhecer que o resultado está correto, completo, devidamente formatado e claramente legível. Isso conclui a tentativa.
        - A7c1) Se o competidor ou o juiz se recusar a aceitar e assinar a súmula, o Delegado da WCA tem que resolver a disputa.

>         - A7c2) Se o competidor assinar intecnionalmente (ou marcar de outra forma) uma tentativa antes que a tenha iniciado, ele perde o direito àquela tentativa e o resultado será considerado DNS.

Reviewer: ```suggestion
        - A7c2) Se o competidor assinar intencionalmente (ou marcar de outra forma) uma tentativa antes que a tenha iniciado, ele perde o direito àquela tentativa e o resultado será considerado DNS.
```

    - B2c) Quando estiver usando o timer Stackmat, o competidor não pode ter contato físico com o quebra-cabeça ou o pote enquanto estiver iniciando a tentativa. Pena: pena de tempo (+2 segundos).
    - B2d) Quando estiver usando o timer Stackmat, o competidor inicia a tentativa removendo suas mãos do timer, iniciando-o. (Isso também inicia a resolução.)
        - B2d1) O competidor remove o pote do quebra-cabeça depois de ter iniciado o timer.
        - B2d2) Se um cronômetro estiver sendo usado além do timer Stackmat, o juiz o inicia no momento em que o competidor inicia a resolução.
    - B2f) Quando um cronômetro estiver sendo usado como único marcador de tempo, o competidor confirma que está pronto e inicia a resolução levantando o pote. O juiz inicia o cronômetro assim que o competidor iniciar a resolução.
- B3) Fase de memorização:
    - B3a) O competidor pode pegar o quebra-cabeça durante a fase de memorização.
    - B3b) O competidor não pode fazer anotações. Pena: desqualificação da tentativa (DNF).

>     - B3c) O competidor não pode aplicar movimentos ou mudanças de alinhamento intentionais (ver [Regra A3c2](regulations:regulation:A3c2)) ao quebra-cabeça durante a fase de memorização.

Reviewer: ```suggestion
    - B3c) O competidor não pode aplicar movimentos ou mudanças de alinhamento intencionais (ver [Regra A3c2](regulations:regulation:A3c2)) ao quebra-cabeça durante a fase de memorização.
```

    - B3d) Se as partes do quebra-cabeça não estiverem completamente alinhadas, então o competidor pode alinhar as faces, contanto que os desalinhamentos estejam dentro dos limites da [Regra 10f](regulations:regulation:10f).
- B4) Fase vendada:
    - B4a) O competidor coloca a venda para iniciar a fase vendada.
        - B4a1) Até que o competidor aplique o primeiro movimento ao quebra-cabeça, ele pode remover a venda e retornar à fase de memorização.

>     - B4b) O competidor não pode aplicar movimentos ou mudanças de alinhamento intentionais ao quebra-cabeça antes de ter colocado totalmente a venda. Pena: desqualificação da tentativa (DNF).

Reviewer: ```suggestion
    - B4b) O competidor não pode aplicar movimentos ou mudanças de alinhamento intencionais ao quebra-cabeça antes de ter colocado totalmente a venda. Pena: desqualificação da tentativa (DNF).
```

    - B4c) Assim que o competidor aplicar o primeiro movimento no quebra-cabeça, o juiz precisa garantir que há um bloqueador de visão (p. ex. uma folha de papel ou papelão) entre o quebra-cabeça e o rosto do competidor enquanto ele estiver resolvendo.
        - B4c1) Em todos os casos, o competidor precisa usar a venda de forma que sua visão do quebra-cabeça ainda estaria claramente bloqueada se o bloqueador de visão não estivesse no caminho.
        - B4c3) Se o juiz e o competidor concordarem anteriormente, o competidor pode escolher posicionar ele mesmo o quebra-cabeça atrás de um objeto apropriado (p. ex. uma estante de partitura, a superfície da mesa) durante a fase vendada.
    - B4d) O competidor não pode ver o quebra-cabeça em nenhum momento durante a fase vendada. Pena: desqualificação da tentativa (DNF).
- B5) Finalizando a resolução:
    - B5a) Quando estiver usando o timer Stackmat, o competidor finaliza a tentativa soltando o quebra-cabeça e parando o timer.

>         - B5a1) Se um cronômetro de mão estiver sendo usado em adição ao timer Stackmat, o juíz para o cronômetro de mão assim que o competidor para a resolução.

Reviewer: ```suggestion
        - B5a1) Se um cronômetro de mão estiver sendo usado em adição ao timer Stackmat, o juiz para o cronômetro de mão assim que o competidor para a resolução.
```

        - E2c5) A solução do competidor é considerada correta se começando com um quebra-cabeça resolvido e aplicando a sequência de embaralhamento seguida pela solução o quebra-cabeça terminar resolvido. Pena para uma solução incorreta: desqualificação da tentativa (DNF).
    - E2d) O tamanho da solução é calculado usando-se a "Outer Block Turn Metric" (ver [Regra 12a](regulations:regulation:12a)).
        - E2d1) O tamanho máximo permitido para a solução é de 80 (movimentos e rotações).
    - E2e) A solução do competidor não pode ser diretamente derivada de qualquer parte da sequência de embaralhamento. Pena: desqualificação da tentativa (DNF).
        - E2e1) O Delegado da WCA pode pedir ao competidor que explique o propósito de cada movimento em sua solução, independentemente da sequência de embaralhamento. Se o competidor não puder dar uma explicação válida, a tentativa é desqualificada (DNF).
- E3) O competidor pode usar os seguintes objetos durante a tentativa. Pena por usar objetos não-autorizados: desqualificação da tentativa (DNF).

>     - E3a) Papel (fornecido pelo juiz), a folha de solução (fornecida pelo juíz), uma foto do embaralhamento para aquela tentativa (que pode ser inclusa no paper ou na folha de resolução) e canetas/lápis ou similar (fornecido pelo juiz, ou opcionamente auto-fornecido).

Reviewer: ```suggestion
    - E3a) Papel (fornecido pelo juiz), a folha de solução (fornecida pelo juiz), uma foto do embaralhamento para aquela tentativa (que pode ser inclusa no paper ou na folha de resolução) e canetas/lápis ou similar (fornecido pelo juiz, ou opcionalmente auto-fornecido).
```

    - E3b) Cubos 3x3x3 (no máximo 3, fornecidos pelo competidor), conforme descrito no [Artigo 3](regulations:regulation:3).
    - E3c) Adesivos coloridos (fornecidos pelo competidor).
    - E3d) Cronômetro ou relógio (fornecido pelo competidor) para acompanhar o tempo decorrido, se aprovado pelo Delegado da WCA.
    - E3e) Outros aparelhos eletrônicos que não dêem uma vantagem injusta, a critério do Delegado da WCA (ver [Regra 2i1](regulations:regulation:2i1)).
    - E3f) Ferramentas para remover marcações (fornecidas pelo competidor), como líquido corretivo, fita corretiva ou borrachas.
- E4) O Delegado da WCA pode optar por publicar as soluções (p. ex. foto da súmula ou transcrição das soluções).

> - E5) O competidor não pode se comunicar com ninguém além do Delegado da WCA ou do juíz. Esta comunicação deve ser sobre Regras da WCA e o competidor não pode receber assistência em sua solução. Penalidade: desqualificação da tentativa (DNF).

Reviewer: ```suggestion
- E5) O competidor não pode se comunicar com ninguém além do Delegado da WCA ou do juiz. Esta comunicação deve ser sobre Regras da WCA e o competidor não pode receber assistência em sua solução. Penalidade: desqualificação da tentativa (DNF).
```
